File: sdks/java/lib/src/main/java/models/shared/RecentItems.java
package .models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * RecentItems
 * The part of an API response that describes marker
 * based pagination
**/
public class RecentItems {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("entries")
    public RecentItem[] entries;
    public RecentItems withEntries(RecentItem[] entries) {
        this.entries = entries;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("limit")
    public Long limit;
    public RecentItems withLimit(Long limit) {
        this.limit = limit;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("next_marker")
    public Long nextMarker;
    public RecentItems withNextMarker(Long nextMarker) {
        this.nextMarker = nextMarker;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("prev_marker")
    public Long prevMarker;
    public RecentItems withPrevMarker(Long prevMarker) {
        this.prevMarker = prevMarker;
        return this;
    }
}

File: python-sdk/src/boxsdk/models/shared/recentitems.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils
from ..shared import recentitem as shared_recentitem


@dataclass_json
@dataclasses.dataclass
class RecentItems:
    r"""RecentItems
    The part of an API response that describes marker
    based pagination
    """
    
    entries: Optional[list[shared_recentitem.RecentItem]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    prev_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prev_marker') }})
    
